### PR TITLE
Remove animation from donut chart for icon entry points

### PR DESCRIPTION
**Pull request by gabycampagna**
Remove animation from donut chart for icon entry points

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 13189ae93c89715ffc376b3d9dbbe306b8ebadfa

https://deploy-preview-2929--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
I'm not sure if you already tried this - is it possible to just have both animation happen at the same time?  Does that remove the flicker in the icon?  

If not then I think the chart should detect if it is an icon based on the height / width passed in. My concern is that passing the `icon` value all up and down the chart components is probably going to be annoying in the long run.  It would make more sense for the donut chart to decide that at anything less than a size 100x100 for example) some animations don't run.  And maybe the pie chart makes a different decision at a different break point.

**Comment by gabycampagna** [comment]
I just tried the animation running at the same time and the flicker is still there. The labels are also displaying and is cluttering up the icon.

**With labels rendering:**
screen shot 2017-04-30 at 12 50 42 pm

No labels: 
screen shot 2017-04-30 at 12 49 43 pm

I understand what you are saying about passing the ```icon``` value on all components in the long run. 

I'll see if I can figure out how the solution you have requested.

File: app/components/chart-donut.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ export default Component.extend({
   data: null,
   width: null,
   height: null,
+  icon: false,
```
Comment (jrjohnson) [inline app/components/chart-donut.js]: Old code. Should be removed

Comment (gabycampagna) [inline app/components/chart-donut.js]: Awe! Ok!

**Pull request merged**